Source/Serilog.Exceptions/Serilog.Exceptions.csproj

```diff
@@ -0,0 +1,48 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup Label="Build">
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TargetFrameworks>net45;netstandard1.3;netstandard1.6</TargetFrameworks>
+    <CodeAnalysisRuleSet>../../MinimumRecommendedRulesWithStyleCop.ruleset</CodeAnalysisRuleSet>
+    </PropertyGroup>
+
+  <PropertyGroup Label="Package">
+    <Description>Log exception details and custom properties that are not output in Exception.ToString().</Description>
+    <AssemblyTitle>Serilog Exceptions</AssemblyTitle>
```

Comment:
Only need to specify the following here (Copied from another project of mine):

```
<VersionPrefix>1.0.1</VersionPrefix>
    <Authors>Muhammad Rehan Saeed (RehanSaeed.com)</Authors>
    <Product>Boilerplate Framework</Product>
    <Description>Provides boilerplate framework code for a .NET project.</Description>
    <Copyright>Copyright © Muhammad Rehan Saeed. All rights Reserved</Copyright>
    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
    <PackageLicenseUrl>https://github.com/ASP-NET-MVC-Boilerplate/Framework/blob/master/LICENSE</PackageLicenseUrl>
    <PackageProjectUrl>https://github.com/ASP-NET-MVC-Boilerplate/Framework</PackageProjectUrl>
    <PackageIconUrl>https://raw.githubusercontent.com/RehanSaeed/ASP.NET-MVC-Boilerplate/master/Images/Icon.png</PackageIconUrl>
    <RepositoryUrl>https://github.com/ASP-NET-MVC-Boilerplate/Framework.git</RepositoryUrl>
    <RepositoryType>git</RepositoryType>
    <PackageTags>Boilerplate;Muhammad Rehan Saeed;Framework</PackageTags>
    <PackageReleaseNotes>Initial version.</PackageReleaseNotes>
```

I move the signing properties to a separate section (Copied from another project):

```
<PropertyGroup Label="Signing">
    <SignAssembly>true</SignAssembly>
    <AssemblyOriginatorKeyFile>../../../Key.snk</AssemblyOriginatorKeyFile>
    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
  </PropertyGroup>
```